Patch104pZH/GameFilesEdited/Data/INI/Object/ChinaVehicle.ini

```
@@ -458,74 +452,170 @@ Object ChinaTankOverlordGattlingCannon
   ;UpgradeCameo4 = NONE
   UpgradeCameo5 = Upgrade_ChinaOverlordGattlingCannon
 
+  ; Patch104p, NProject @bugfix properly implement Gattling Cannon to show muzzle flashes and smoke particles
```

Comment should be neutral. The previous implementation is just as "proper".
```suggestion
  ; Patch104p, NProject @bugfix Implement Gattling Cannon muzzle flashes and smoke particles
```